File: Python/INICIANTE/1962 Há Muito, Muito Tempo Atrás.py
testes = int(input())
while testes != 0:
    ano = int(input())
    if ano >= 2015:
        print("%d A.C." % (ano - (2015-1)))
    else:
        print("%d D.C." % (2015 - ano))
    testes -= 1

File: C/BEGINNER/1962 Há Muito, Muito Tempo Atrás.c
#include <stdio.h>

int main() {
    int testes, ano;
    scanf("%d", &testes);
    while (testes != 0)
    {
        scanf("%d", &ano);
        if (ano >= 2015)
        {
            printf("%d A.C.\n", (ano - (2015-1)));
        }
        else
        {
            printf("%d D.C.\n", (2015-ano));
        }
        testes -= 1;
    }
    return 0;
}
